From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brad Miller <brad@wpi.edu>
Date: Wed, 16 Apr 2014 15:20:52 -0400
Subject: [PATCH 0038/6262] Fix 2 path problems in some scripts

Fix build.xml to use the correct path for ar that is used to build the
libraries package when the version updates. Also update the runjavaprogram
script to run java from the /usr/local/frc/JRE directory.

Change-Id: Ifbc60e4975ca7e23f05cfcc92d3b3ab0d695b5e6
---
 .../src/main/resources/cpp-zip/ant/build.xml                   | 2 +-
 .../src/main/resources/cpp-zip/ant/runjavaprogram              | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
index 166ea15bb80f3ae539c4b3c0cd8c6d7aaacc7fcc..3627d1784bb74763883106aeacfd79c37936cb25 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
@@ -109,7 +109,7 @@ ${md5.hal} usr/local/frc/lib/libHALAthena.so</echo>
 				</tarfileset>
 			</tar>
 			<gzip destfile="${wpilib.ant.dir}/control.tar.gz" src="${wpilib.ant.dir}/control.tar"/>
-			<exec command="arm-none-linux-gnueabi-ar -r ${opkg.name}_${cpp-sos}_${opkg.arch}.deb debian-binary control.tar.gz data.tar.gz" dir="${wpilib.ant.dir}"/>
+			<exec command="~\wpilib\toolchains\arm-none-linux-gnueabi-4.4.1\bin\arm-none-linux-gnueabi-ar -r ${opkg.name}_${cpp-sos}_${opkg.arch}.deb debian-binary control.tar.gz data.tar.gz" dir="${wpilib.ant.dir}"/>
 		    <scp file="${wpilib.ant.dir}/${opkg.name}_${cpp-sos}_${opkg.arch}.deb" todir="${username}@${target}:/tmp"
 				 password="${password}" trust="true"/>
 			<sshexec host="${target}"
diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/runjavaprogram b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/runjavaprogram
index 114b837be9a28cccc2643bc59702b307024fcbd6..272b38cd183cf808a96d69c6fd1aa6e52e7ab869 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/runjavaprogram
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/runjavaprogram
@@ -2,4 +2,5 @@
 killall java
 killall FRCUserProgram
 sleep 1
-nohup java -Djna.library.path=$LD_LIBRARY_PATH -Xmx32M -jar FRCUserProgram.jar
+nohup /usr/local/frc/JRE/bin/java -jar FRCUserProgram.jar
+#nohup /usr/local/frc/JRE/bin/java -Djna.library.path=$LD_LIBRARY_PATH -Xmx32M -jar FRCUserProgram.jar
